bgpd/bgp_routemap.c

```diff
@@ -4058,7 +4058,7 @@ DEFUN (set_aspath_exclude,
 
 DEFUN (no_set_aspath_exclude,
        no_set_aspath_exclude_cmd,
-       "no set as-path exclude (1-4294967295)...",
+       "no set as-path exclude [(1-4294967295)]",
```

Comment:
Can't do it this way, breaks entering more than one AS which makes it impossible to deconfigure a rule that has more than one AS

Reply:
ah, really.. I will take a look more deeply into this :+1:

Reply:
Try writing an alias command that doesn't take any parameters and alias it to this one, then update the code for this one to handle not having any arguments